tests/test_complex.py

```diff
@@ -0,0 +1,14 @@
+from lassen.stdlib.fma import gen_FMA
+from lassen.isa import DATAWIDTH
+from hwtypes import BitVector, Bit
+from lassen.sim import gen_pe
+import lassen.asm as asm
+
+Bit = Bit
+Data = BitVector[DATAWIDTH]
+
+FMA = gen_FMA(BitVector.get_family())
+
+def test_fma():
```

Comment:
Shoud do random testing as done in test_pe not just a single set of valyes

Reply:
Agreed, but let me do that in a different PR.

Reply:
This is more a simple infrastructure PR

Reply:
Does it not work with randomized inputs? If so I probably shouldn't be merged else new code should come with testing.

Reply:
Added more parameterized tests.